docs/multicluster/setup-multi-primary.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script sets up a primary-primary multi-network Istio mesh locally with kind using the Sail operator.
+# It is adapted from: https://istio.io/latest/docs/setup/install/multicluster/multi-primary_multi-network/.
+
+if [ -z "$1" ]; then
+  echo "Usage: $0 <absolute-path-to-istioctl-dir>"
```

Comment:
isn't the path that is required the path of the istio repository?

Reply:
When you download istioctl you also download the various samples/tools for that version that are used in this multi-cluster tutorial but these can be referenced outside of this dir and I think it's fine for this script to require you have `istioctl` in your path. I'll update this and remove this requirement.

Reply:
I updated this to download the examples/scripts directly from github rather than rely on them being present in the local istioctl dir. You now need `istioctl` in your path to run the script though.